syntax_and_semantics/annotations.md

```diff
@@ -78,6 +78,39 @@ end
 annotation_value # => 2
 ```
 
+The `named_args` method can be used to read all key/value pairs on an annotation as a `NamedTupleLiteral`.
+
+```crystal
+annotation MyAnnotation
+end
+
+@[MyAnnotation(value: 2, name: "Jim")]
+def annotation_named_args
+  {{@def.annotation(MyAnnotation).named_args}}
+end
+
+puts annotation_named_args # => {value: 2, name: "Jim"}
+```
+
+This can also be used within macros.
```

Comment:
Isn't that obvious? I mean, does this second example provide a meaningful benefit? I see it more as unnecessary clutter.
Alternatively we could scrap the first one and only show this with more stuff, but I doubt that's really necessary.